aqua/barrier/Portfile

```diff
@@ -46,15 +44,33 @@ if { ${os.major} < 16 } {
     configure.args-append \
         -DBARRIER_BUILD_GUI=OFF
 } else {
-    PortGroup  qt5 1.0
-    PortGroup           app 1.0
+    PortGroup   qt5 1.0
+    PortGroup   app 1.0
 
     qt5.depends_component \
         qtbase
```

Comment:
this should be implied by the `qt5` PortGroup and as you see `port lint` throws a warning about it; please remove this and check

Reply:
@reneeotten Removed that line.